metrics/web/lib/common/presentation/tooltip_icon/theme_data/tooltip_icon_theme_data.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
```

Comment:
Should we place this file under the `tooltip_icon/theme/theme_data` package to be more consistent?